src/app.js

```diff
@@ -513,36 +510,58 @@ var run = function () {
     loadVersion($('#versionSelector').val());
   });
 
-  // var soljsonSources is provided by bin/list.js
-  if (window.soljsonSources !== undefined) {
-    // populate selector list with available versions
-    $.each(soljsonSources, function (i, file) {
-      if (file) {
-        var version = file.replace(/soljson-(.*).js/, '$1');
-        $('#versionSelector').append(new Option(version, file));
-      }
+  $.getJSON('https://ethereum.github.io/solc-bin/bin/list.json', function (data, status) {
+    //
+    // loading failed for some reason, fall back to local compiler
+    //
+    if (status !== 'success') {
+      // always include the local version
+      $('#versionSelector').append(new Option('latest local version', 'builtin'));
+
+      loadVersion('builtin');
+      return;
+    }
+
+    //
+    // populate version dropdown with all available compiler versions
+    //
+
+    // Sort builds according to semver
+    var builds = data.builds.sort(function (a, b) {
+      a = a.version + '-' + a.prerelease;
```

Comment:
In the future, `a.prerelease` might be empty. In that case, we also have to drop the `'-'`